tool/dart_dev/config.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:dart_dev/dart_dev.dart';
+
+final Map<String, DevTool> config = {
+  'analyze': AnalyzeTool()..analyzerArgs,
+  'format': FormatTool(),
+  'test': TestTool(),
+  'serve': WebdevServeTool()..buildArgs = ['example'],
+};
```

Comment:
Let's use the `coreConfig` that is provided by dart_dev proper to clean this up:

```suggestion
final Map<String, DevTool> config = {
  ...coreConfig,
  'serve': WebdevServeTool()..buildArgs = ['example'],
};
```

Also, we probably should update the Readme file to indicate that the example can/should be run using the `pub run dart_dev serve` command.

Reply:
Is `coreConfig` Workiva speicific?

Reply:
Nope, it comes from the `dart_dev` package.